### ISSUE TITLE
FR Sort by known  ETV

### ISSUE THREAD

**Issue by RealFakeGitter** [FMaz008/VineHelper#63]
Feature Request: There is already a feature that displays the ETV once at item is clicked, which is great, for those items that the ETV has already been saved for it would be really cool if we can sort the searches based on that ETV low to high etc.

**Comment by Wladefant** [comment]
That seems to be solved as the etv is now automatically fetched. That is even better than what I proposed with using a third party service. @FMaz008 could you share the committee where this is implemented?

**Comment by Wladefant** [comment]
Additionally @FMaz008 the feature where you share the etv is not needed anymore as all etv are provided ![Screenshot_20250118_083942_Yandex Browser.jpg](https://github.com/user-attachments/assets/14fcd77b-ed6c-49f1-862d-77bd71552bf7)


When the etv is automatically fetched the extension does not seem to understand it and fetch from the user

**Comment by Wladefant** [comment]
![20250118_091416.jpg](https://github.com/user-attachments/assets/262c83dd-c7c6-4853-b891-d2a74ed6e493)

All right maybe I was wrong, probably someone created a bot that clicks on see details?

That is why I didn't find the commit, it was really great 🙂. 

I would really propose getting the price through a third party service with an api for the products. They are quite cheap and improves the life of viners substantially instead of always clicking

**Comment by hutauf** [comment]
You are not gonna get etv=0 information through third party, right?

**Comment by Wladefant** [comment]
Yeah that is unfortunat4ly not possible price does not equal etv but until there is not etv price will suffice. That should be of course a different label, a dollar sign or something instead of etv

**Comment by FMaz008** [comment]
Seems I missed this discussion entirely before today. 

In short:
- No the ETV is not automatically fetched.
- No, a 3rd party API would not allow me to get ETV values.
- Yes, a 3rd party API would allow me to get the price for the item, but the cost for the amount of items VH handles would be significant. It's on my list to look into it at some point though.
- As I understand the initial request was asking to sort by ETV, which is now possible in the notification monitor. I'll close this issue.
